inspirehep/config.py

```diff
@@ -596,3 +598,16 @@
 }
 """Updates default properties that should be added to the Holding Pen index
 mappings."""
+
+CACHED_VIEWS = {
+    'single_entry': {
```

Comment:
No, it's a single entry in the `references` or `citations` box in detailed view of the record.